src/Microsoft.AspNetCore.Mvc.Testing/Internal/TestCulture.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Testing.Internal
+{
+    public class TestCulture
+    {
+        public string Culture { get; set; }
```

Comment:
Sorry, I meant the input - `UseRequestCulture(string culture, string uiCulture)` which would result in this being `CultureInfo`.